.travis.yml

```diff
@@ -20,3 +20,5 @@ before_install:
   - src/build/bin/install-samtools.sh
   - src/build/bin/install-bcftools.sh
   - src/build/bin/checkout-gatk-large-files.sh
+after_success:
+  - '[[ $TRAVIS_BRANCH == "master" ]] && { mvn deploy --settings src/build/travis/settings.xml -DskipTests=true -B ; };'
```

Comment:
This will run on every matrix job.  We can restrict it by adding an environment variable to one that we check.  Every pr build will be on the master branch as well.  We have to decide how many snapshots we want to publish and in what circumstances.  I'm an advocate for publishing a snapshot for every branch. 

We also need to consider naming because Ii believe all of these snapshots will have the same name as well which means we'll have competing snapshots overriding each other.

Reply:
Publish a snapshot for every branch?  That sounds very confusing from a client point of view.  In Big Data Genomics projects we CI for every pull request/branch but only publish snapshots from master, after every commit and with the nightly build.

Reply:
I think we should only publish snapshots from master, at least to start with. I've updated the PR to do that (fixing the problem where it publishes for every job in the matrix).

Note that this PR passed tests and *didn't* publish to the snapshot repo (at https://oss.sonatype.org/content/repositories/snapshots/org/disq-bio/disq/), which is what is expected.